src/lune/builtins/process/options.rs

```diff
@@ -133,6 +134,20 @@ impl<'lua> FromLua<'lua> for ProcessSpawnOptions {
             }
         }
 
+        /*
+            If we have stdin contents, we need to pass those to the child process
+        */
+        match value.get("stdin")? {
+            LuaValue::Nil => this.stdin = None,
+            LuaValue::String(s) => this.stdin = Some(s.to_string_lossy().to_string()),
```

Comment:
The lossy conversion is not necessary here, we don't even need to use a `String` actually, we can store a `Vec<u8>` for plain bytes instead

Reply:
Fixed in https://github.com/filiptibell/lune/pull/106/commits/33730e1147f65e9d284a535de8b555959a44d63f.

Reply:
Resolved in https://github.com/filiptibell/lune/pull/106/commits/47d39cb8a9302e6886e4280f978fdc9a9d8d301a!